modules/ROOT/pages/app-declaration.adoc

```diff
@@ -0,0 +1,38 @@
+= Application declaration 
+:description: Application declaration 
+
+In Bonita Central, we refer as application to the entity that will be built and run by Bonita. It is the content of a unique Git repository. 
+One Runtime will be able to host several applications that share the same Business Data Model. 
+
+== Prerequisites
+. All Runtimes (nodes) linked to an application space should have the same version. 
+. To facilitate the data exchange, install Bonita Central in the same network as your Bonita Runtimes. If not, authorization to the network will be required. 
+. Have Bonita Runtime admin credentials 
+
+== How to declare an application ? 
+
+After installing Bonita Central, the first thing you will have to do is adding the applications you want to supervise. 
+Nothing easier because you will be able to do it directly from the overview page. 
+
+The only information you will have to provide is : 
+
+* A name and a description for the application. Make it transparent both for technical and business users 
+* Runtime end point - URL, IP adress - to access it 
+* Connection credentials - Bonita Runtime admin username and password. This information will not be stored in Bonita Central. It will only be used to establish a connection with the engine.
```

Comment:
Speak about a cluster deployment: in this case the connection should be on the load balancer.

Reply:
suggestion included.